Application/Extensions/SpreadsheetExtensions.cs

```diff
@@ -24,6 +24,11 @@ public static decimal ParseDecimal(this string? cellValue)
         return 0;
     }
 
+    public static decimal ParsePrice(this string? cellValue, decimal vatPercentageToApply)
+    {
+        return Math.Round(cellValue.ParseDecimal() * (1 + (vatPercentageToApply / 100)), 2);
```

Comment:
I wonder if this should be `Math.floor` so that we always round down?

Reply:
It's an interesting one...  

Going forward the prices will be supplied in the spreadsheet exc of VAT.  As this code suggests, for now we will apply VAT on the spreadsheet values that we store in the database (may change in the future with other VAT changes).  When doing this move to the new system there were some rounding errors, so for now the exclusive figure that is being supplied in the spreadsheets is going to be supplied in more than 2 decimal places, so the existing prices should convert back to an inclusive value without any rounding needed.

So the question is that going forward as new exclusive prices are added by the TPs what should happen when VAT is added in terms of rounding.  Checking the following it looks like rounding is the preferred method rather than floor: https://www.gov.uk/hmrc-internal-manuals/vat-trader-records/vatrec12030#:~:text=If%20the%20VAT%20on%20any,it%20should%20be%20rounded%20up.